.github/workflows/image-factory.yaml

```diff
@@ -297,12 +297,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: securesign/cosign
-          ref: ${{ env.COSIGN_VER }}
-
-      - name: Pull down the securesign repo
-        uses: actions/checkout@v2
-        with:
-          path: securesign
+          ref: release-next
```

Comment:
Once this lands https://github.com/securesign/cosign/pull/1#pullrequestreview-1595758085 and we run the update script as `./redhat/release/update-to-head.sh v2.1.1` to create a `redhat-v2.1.1` branch, this PR should be good to go. But the refs should be to the `redhat-v2.1.1` branch, which can be set in `COSIGN_VER`.